packetbeat/protos/cassandra/config.go

```diff
@@ -0,0 +1,27 @@
+package cassandra
+
+import (
+	"github.com/elastic/beats/packetbeat/config"
+	"github.com/elastic/beats/packetbeat/protos"
+)
+
+type cassandraConfig struct {
+	config.ProtocolCommon `config:",inline"`
+	SendRequestHeader     bool   `config:"send_request_header"`
+	SendResponseHeader    bool   `config:"send_response_header"`
+	Compressor            string `config:"compressor"`
+}
+
+var (
+	defaultConfig = cassandraConfig{
+		ProtocolCommon: config.ProtocolCommon{
+			TransactionTimeout: protos.DefaultTransactionExpiration,
+		},
+		SendRequestHeader:  false,
+		SendResponseHeader: false,
+	}
+)
+
+func (c *cassandraConfig) Validate() error {
+	return nil
```

Comment:
check here if Compressor is valid (empty or 'snappy'). This will generate a somewhat readable error message pointing to the config option and config file (or CLI overwrite) that is wrong.